src/main/java/io/github/coolmineman/cheaterdeleter/modules/packetcount/PacketLimiterCheck.java

```diff
@@ -21,6 +21,22 @@ public PacketLimiterCheck() {
     public ActionResult onPacket(CDPlayer player, Packet<ServerPlayPacketListener> packet) {
         if (!enabledFor(player)) return ActionResult.PASS;
         PacketVolumeData data = player.getOrCreateData(PacketVolumeData.class, PacketVolumeData::new);
+
+        if (packet.getClass().getName().equals("net.minecraft.class_2840")) {
+            if (data.craftingCount != 0 ) {
+                if (System.currentTimeMillis() - data.lastCraft < 125) {
```

Comment:
this is the only reason i can think of why the crasher doesnt get caught sometimes, other than a really inefficient looping of the limiter itself... (which I have no clue how to debug tbh)